### ISSUE TITLE
Explain the baseline adjustment in a short example

### ISSUE THREAD

**Issue by DCEW** [The-Strategy-Unit/nhp_inputs#703]
Maybe in the explanatory box, add something like:
"This adjustment accounts for activity numbers (admission or attendance) that maybe under or over counted at the hospital trust. It relates to Hospital Episodes Statistics (HES) data (from providers) via Secondary Users Service (SUS). The adjustment required is the Percentage adjustment to the baseline data currently extracted for your trust for each POD/specialty.

For example, if our baseline data showed you had 10,000 general surgery admissions, but your local data shows you had 12,000, you may want to add in these 2000 'missing' admissions before modelling.  To do this, you could move the slider beside the 'General Surgery' heading to the right until the label in blue reads '2000'. The 'relative change' then shows you the multiplier which will be used to increase the counts of this activity type in the  data. In this instance this would be 12,000 / 10,000 = 1.2.

Please provide the reason for your adjustment and cite an evidence base.

**Comment by yiwen-h** [comment]
Note that the baseline adjustment does not change the data "before modelling" - it is just another change factor that is multiplied together with all the other change factors to create the summary parameter _lambda_
